files/en-us/web/api/request/index.html

```diff
@@ -27,6 +27,10 @@ <h2 id="Constructor">Constructor</h2>
 <h2 id="Properties">Properties</h2>
 
 <dl>
+ <dt>{{domxref("Request.body")}} {{readonlyInline}}</dt>
+ <dd>A {{domxref("ReadableStream")}} of the body contents.</dd>
+ <dt>{{domxref("Request.bodyUsed")}} {{readonlyInline}}</dt>
+ <dd>Stores a {{domxref("Boolean")}} that declares whether the body has been used in a response yet.</dd>
```

Comment:
```suggestion
 <dd>Stores a {{domxref("Boolean")}} that declares whether the body has been used yet.</dd>
```

I'm actually not sure if this is right, but "response" was wrong.